README.md

```diff
@@ -569,7 +567,52 @@ function App() {
 }
 ```
 
-For a comprehensive list of options you can provide to the `useStoryblokRichText`, please consult the [Full options](https://github.com/storyblok/richtext?tab=readme-ov-file#options) documentation.
+### Server-side Rich Text Rendering
+
+> [!INFO]  
+> Recommended for Next.js 15 and React 19.
+
+You can render rich text fields by using the `StoryblokServerRichText` component:
+
+```ts
+import { StoryblokServerRichText, useStoryblok } from '@storyblok/react/rsc';
+
+function App() {
+  const story = useStoryblok('home', { version: 'draft' });
+
+  if (!story?.content) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div>
+      <StoryblokServerRichText doc={story.content.richText} />
+    </div>
+  );
+}
```

Comment:
On this example we are using the `rsc` module which is meant for Next.js App Router, but the code does not reflect the changes in the DX (they are documented in the README).

So either we change the example detailing that this is for App Router and reuse the same code, or we document that this is a non-strict example that only details how to use the `StoryblokServerRichText` component but everything else needs to be adapted to the user's context.